test/unit/constants.dart

```diff
@@ -311,7 +310,9 @@ final List<String> draft2019SkippedTestFiles = [
 
 final List<String> draft2019SkippedTests = [
   // Requires unevaluatedProperties support
-  'ref creates new scope when adjacent to keywords : referenced subschema doesn\'t see annotations from properties'
+  'ref creates new scope when adjacent to keywords : referenced subschema doesn\'t see annotations from properties',
+  'unevaluatedItems with \$ref : with no unevaluated items',
+  'unevaluatedItems with \$ref : with unevaluated items',
```

Comment:
The ref tests for this keyword lead me to believe `$ref` is currently implemented incorrectly and should be fixed in some follow up work.

Reply:
![ok](https://media.giphy.com/media/Y5PnpRvm8PVba/giphy.gif)